src/libaktualizr/crypto/crypto.cc

```diff
@@ -322,6 +322,7 @@ bool Crypto::extractSubjectCN(const std::string &cert, std::string *cn) {
 
 /**
  * Generate a RSA keypair
+ * @param key_type Type of key (RSA2048, RSA4096, etc.)
```

Comment:
Can we actually enumerate the possible values here, instead of "etc."? I bet there are few enough that it's reasonable, right?

Reply:
Yes, that is possible, although I would argue it would be better to go to the actual definitions directly instead of duplicating the information here. (Also, this was just my quick attempt to clean up some doxygen complaints -- this isn't part of the API that I wanted to work on.)

Reply:
Ah, gotcha. And I think I agree that it's better not to.